lib/src/cardano_ledger_application.dart

```diff
@@ -30,7 +30,7 @@ import 'package:ledger_cardano/src/utils/validation_exception.dart';
 import 'package:ledger_flutter/ledger_flutter.dart';
 
 class CardanoLedgerApp {
-  static bool debugPrintEnabled = false;
+  static bool debugPrintEnabled = true;
```

Comment:
This should stay as `false` here. You can modify this to `true` **at runtime** inside the tests/etc by calling `CardanoLedgerApp.debugPrintEnabled = true`.

Reply:
done